src/scores/processing.py

```diff
@@ -0,0 +1,69 @@
+"""Tools for processing data for verification"""
+from typing import Union
+
+import xarray as xr
+
+
+def broadcast_and_match_nan(*args: Union[xr.DataArray, xr.Dataset]):
```

Comment:
I initially had this in `scores.functions` but created a new file called `scores.processing` so that it has a more specific file name. This function will be used a lot by people pre-processing data before verification (e.g., matching forecast pairs). There will be some other pre-processing that we migrate across, such as tools to discretise data based on a threshold.

Thoughts? I am fine with either option.

Reply:
I'm happy with that explanation and happy to leave it as processing.